src/cli/tls_http_server.cpp

```diff
@@ -12,11 +12,11 @@
 
 #include "cli.h"
 
-#if defined(BOTAN_HAS_BOOST_ASIO)
-   #include <botan/boost_compat.h>
+#if defined(BOTAN_HAS_TLS_ASIO_STREAM)
+   #include <botan/asio_compat.h>
 #endif
```

Comment:
This checked for the wrong module (the meta-module `boost` that also used to contain `boost_compat.h`). It is available when configured `--with-boost`. Downstream in this file we didn't check for `BOTAN_HAS_TLS_ASIO_STREAM` but assumed that, if `BOTAN_FOUND_COMPATIBLE_BOOST_VERSION` was set, that `asio_stream.h` would be available. This is not true when configured with `--disable-modules=asio`, though.